lms/templates/problem_notifications.html

```diff
@@ -4,9 +4,10 @@
 from openedx.core.djangolib.markup import HTML
 %>
 
-<div class="notification ${notification_type}" tabindex="-1">
+<div class="notification ${notification_type} ${'notification-hint' if hint_notification else 'notification-response'}" tabindex="-1">
```

Comment:
The reason for that is because we can not use /notification to focus. If hint notification shows up, after Submit, the focus goes to submit notification, not notification-hint. Hence, notification-response is used for focus.